xmtp_mls/src/groups/device_sync.rs

```diff
@@ -361,6 +377,75 @@ where
     }
 }
 
+pub struct MessageExpirationWorker<ApiClient, V> {
```

Comment:
we may as well put this in its own file, `groups/expiration_worker.rs` to keep things nice and separated